src/lib.rs

```diff
@@ -114,10 +114,10 @@ impl SmolStr {
         self.0.as_str()
     }
 
-    #[inline(always)]
-    pub fn to_string(&self) -> String {
-        self.as_str().to_string()
-    }
+    // #[inline(always)]
+    // pub fn to_string(&self) -> String {
+    //     self.as_str().to_string()
+    // }
```

Comment:
this probably should be left untouched :)

Reply:
oh shit

Reply:
crimes are now hidden